test/e2e/framework/kubernetes/check-pod-status.go

```diff
@@ -85,3 +78,21 @@ func WaitForPodReady(ctx context.Context, clientset *kubernetes.Clientset, names
 	}
 	return nil
 }
+
+func CheckPodRestarts(ctx context.Context, clientset *kubernetes.Clientset, namespace, labelSelector string) error {
```

Comment:
I think the name is a little misleading. You are checking for container restarts. Can we make the function name more verbose, like `CheckContainerRestart`?